webpack-base.config.js

```diff
@@ -6,9 +6,7 @@ let defineValues = {};
 
 // Expose environment variables
 const envValues = {
-	RIVET_PARTY_API_URL: process.env.RIVET_PARTY_API_URL || null,
-	RIVET_IDENTITY_API_URL: process.env.RIVET_IDENTITY_API_URL || null,
-	RIVET_MATCHMAKER_API_URL: process.env.RIVET_MATCHMAKER_API_URL || null,
+	RIVET_API_ENDPOINT: process.env.RIVET_PARTY_API_URL || null,
```

Comment:
Should this be: `RIVET_API_ENDPOINT: process.env.RIVET_API_ENDPOINT || null,` instead of `RIVET_PARTY_API_URL`?